controller/app/cmdline/src/cmd_line.cpp

```diff
@@ -1015,10 +1043,11 @@ int cmd_line::cmd_view_media_clock(int total_matched, std::vector<cli_argument*>
                 if (stream_output_desc)
                 {
                     avdecc_lib::stream_output_descriptor_response *stream_output_resp_ref = stream_output_desc->get_stream_output_response();
+                    avdecc_lib::descriptor_response_base *desc_base_resp = stream_output_desc->get_descriptor_response();
                     is_clock_sync_source_set = stream_output_resp_ref->stream_flags_clock_sync_source();
                     if(is_clock_sync_source_set)
                     {
-                        desc_obj_name = stream_output_resp_ref->object_name();
+                        desc_obj_name = desc_base_resp->object_name();
```

Comment:
Don't think this edit should be required?
Doesn't stream_outupt_descriptoer_response derive from descriptor_response_base, so should be able to call ->object_name on the derived class?